adminSiteClient/EditorReferencesTab.tsx

```diff
@@ -1,13 +1,139 @@
+                    <li key={post.id} className="list-group-item">
+                        <a href={post.url} target="_blank" rel="noopener">
+                            <strong>{post.title}</strong>
+                        </a>{" "}
+                        (
+                        <a
+                            href={`https://owid.cloud/wp/wp-admin/post.php?post=${post.id}&action=edit`}
+                            target="_blank"
+                            rel="noopener"
+                        >
+                            Edit
+                        </a>
+                        )
+                    </li>
+                ))}
+            </ul>
+        </>
+    ) : (
+        <></>
+    )
+    const gdocsPosts = props.references?.postsGdocs?.length ? (
+        <>
+            <p>Public gdocs pages that embed or reference this chart:</p>
+            <ul className="list-group">
+                {props.references.postsGdocs.map((post) => (
+                    <li key={post.id} className="list-group-item">
+                        <a href={post.url} target="_blank" rel="noopener">
+                            <strong>{post.title}</strong>
+                        </a>{" "}
+                        (
+                        <a
+                            href={`/admin/gdocs/${post.id}`}
+                            target="_blank"
+                            rel="noopener"
+                        >
+                            Edit
+                        </a>
+                        )
+                    </li>
+                ))}
+            </ul>
+        </>
+    ) : (
+        <></>
+    )
+    const explorers = props.references?.explorers?.length ? (
+        <>
+            <p>Explorers that reference this chart:</p>
+            <ul className="list-group">
+                {props.references.explorers.map((explorer) => (
+                    <li key={explorer} className="list-group-item">
+                        <a
+                            href={`https://ourworldindata.org/explorers/${explorer}`}
+                            target="_blank"
+                            rel="noopener"
+                        >
+                            <strong>{explorer}</strong>
+                        </a>{" "}
+                        (
+                        <a
+                            href={`/admin/explorers/${explorer}`}
+                            target="_blank"
+                            rel="noopener"
+                        >
+                            Edit
+                        </a>
+                        )
+                    </li>
+                ))}
+            </ul>
+        </>
+    ) : (
+        <></>
+    )
+    const legacySdgCharts = props.references?.legacySdgCharts?.length ? (
+        <>
+            <React.Fragment>
+                <p>
+                    Legacy SDG referenced charts (slugs of charts only, links to
+                    precise SDG page are not available):
```

Comment:
I can generate a list of slugs together with the article they appear, shouldn't be too much work.

Reply:
I don't think this is needed at this point - we'll migrate away soon anyhow